utility/src/main/java/org/oppia/util/parser/HtmlParser.kt

```diff
@@ -45,7 +45,8 @@ class HtmlParser private constructor(
       )
       htmlContent = htmlContent.replace(
         CUSTOM_IMG_FILE_PATH_ATTRIBUTE,
-        REPLACE_IMG_FILE_PATH_ATTRIBUTE, /* ignoreCase= */false
+        REPLACE_IMG_FILE_PATH_ATTRIBUTE,
+        /* ignoreCase= */false
```

Comment:
space missed `/* ignoreCase= */ false` else everything perfect

Reply:
Oops sorry I missed that! Done now

Reply:
Awesome